src/tools/mmaps_generator/MapBuilder.cpp

```diff
@@ -1016,6 +1022,11 @@ namespace MMAP
         config.detailSampleDist = config.cs * 16;
         config.detailSampleMaxError = config.ch * 1;
 
+        if (isContinentMap(mapID))
+        {
+            config.walkableClimb = m_bigBaseUnit ? 2 : 4;
```

Comment:
@Killyana how about I revert any mmaps_generator change and just push the charge fix that seems to be an improvement over current 3.3.5 branch ? we can continue research about angle and slope later on

Reply:
Yes sounds good.